src/visual/aria.ts

```diff
@@ -82,15 +82,10 @@ export default function ariaVisual(ecModel: GlobalModel, api: ExtensionAPI) {
                 if (ecModel.isSeriesFiltered(seriesModel)) {
                     return;
                 }
-                if (zrUtil.isFunction(seriesModel.enableAriaDecal)) {
```

Comment:
Do not remove these lines. You can remove the `enableAriaDecal` in `SunburstSeries.ts`.